homeworks/vadym.myroshnyk_VadimMir/homework_7/homework_7.js

```diff
@@ -0,0 +1,19 @@
+// fibonacci
+const randomNum = Math.floor(Math.random() * (100 - 50 + 1) + 50);
+
+function fib(random) {
+    const arr = [];
+    let a = 1;
+    let b = 0;
+    for (let i = 0; i <= random; i += a) {
```

Comment:
first element of your array should be 0

Reply:
also you need to check if number is still less then max allowed value, and you check if iterations count is less then max